src/core/SampleBuffer.cpp

```diff
@@ -378,10 +378,10 @@ void SampleBuffer::normalizeSampleRate( const sample_rate_t _src_sr,
 							bool _keep_settings )
 {
 	// do samplerate-conversion to our default-samplerate
```

Comment:
```suggestion
	// do samplerate-conversion to our processing-samplerate
```

Reply:
On second thought, this comment should be changed, unless this line was changed by mistake.